qemu/tests/cfg/pvpanic_event_check.cfg

```diff
@@ -14,4 +14,7 @@
     i440fx:
         expected_cap = 1
     q35:
-        expected_cap = 3
```

Comment:
Do we need set a default value for it?I am not sure if this test case only run on RHEL9/10,if not, we should set a default for other versions of OS.

Reply:
For this case, we are only considering RHEL9 and RHEL10 tests.